voctogui/lib/videodisplay.py

```diff
@@ -163,7 +176,9 @@ def on_error(self, bus, message):
         self.log.debug('Error-Details: #%u: %s', error.code, debug)
 
     def on_level(self, bus, msg):
-        if msg.src.name != 'lvl':
+        p = re.compile('(lvl_)([0-9])')
```

Comment:
`msg.src.name.startswith('lvl_')` should do the trick without an regex: https://docs.python.org/3/library/stdtypes.html#str.startswith